src/definition/livechat/ILivechatTransferData.ts

```diff
@@ -0,0 +1,8 @@
+import { IRoom } from '../rooms';
+import { IUser } from '../users';
+
+export interface ILivechatTransferData {
+    currentRoom: IRoom;
```

Comment:
Should this be `ILivechatRoom` instead of `IRoom`? (I don't know what this all does, so that's why I'm asking)